src/db.js

```diff
@@ -1,4 +1,87 @@
 // @flow
 import store from "react-native-simple-store";
 
-export default store;
+const ACCOUNTS_KEY = "accounts";
+const ACCOUNTS_DB_PREFIX = "accounts.active.";
+
+export default class DB {
+  static get(key: string | any[]): Promise<any> {
+    /** handle accounts case */
+    if (key === ACCOUNTS_KEY) return DB.getAccounts();
+    return store.get(key);
+  }
+
+  static save(key: string | any[], value: any): Promise<any> {
+    /** handle accounts case */
+    if (key === ACCOUNTS_KEY) return DB.saveAccounts(value);
+    return store.save(key, value);
+  }
+
+  static update(key: string, value: any): Promise<any> {
+    return store.update(key, value);
+  }
+
+  static keys(): Promise<any> {
+    return store.keys();
+  }
+
+  static delete(key: string | any[]): Promise<any> {
+    return store.delete(key);
+  }
+
+  static push(key: string, value: any): Promise<any> {
+    return store.push(key, value);
+  }
+
+  static getAccountDbKey = (id: string): string => `${ACCOUNTS_DB_PREFIX}${id}`;
+
+  /** save accounts method between SQLite db and redux store persist */
+  static saveAccounts({
```

Comment:
this needs to happen in an atomic way. there are many ways it could go wrong if multiple saveAccounts happen simultaneously.